packages/massa-proto-cli/README.md

```diff
@@ -0,0 +1,95 @@
+# Massa-Proto-CLI
+
+Massa-Proto-CLI is a tool designed to generate callable functions for smart contract methods in TypeScript and AssemblyScript.
+These generated functions can be seamlessly integrated within your smart contracts from TypeScript or AssemblyScript projects.
+
+## Getting Started
+
+### Prerequisites
```

Comment:
You should add that people should be familirized with:
- massa smart contract dev and deployment
- PB transformer

Reply:
Why do they need to be familiarized with PB transformer ? 
I thought we did all this for them to don"t have to worry about them

Reply:
because event if everything is hidden under the hood, PB is still used to interact with the SC. Maybe familiarized is not the right adjective, but you have the idea.